index.html

```diff
@@ -3419,6 +3440,9 @@ <h3><dfn>Writing content</dfn></h3>
             browsing context</a>, then reject |p| with and
             {{"InvalidStateError"}} {{DOMException}} and return |p|.
           </li>
+          <li>
+            Attempt to <a>abort a pending scan operation</a>.
```

Comment:
Same here, we should return |p| with rejected promise if failing.

Reply:
No, we just reject the pending scan, not this one. It's a cleanup operation.
Or do you mean if the abort fails, we should reject the scan?
i.e. should we handle the "ongoing operation" step as a fail to abort and propagate it up?